### PR TITLE
Build EnMasse Artemis Image

### PR DESCRIPTION
**Pull request by k-wall**

### CODE REVIEW

**Comment by k-wall** [comment]
Needs CI integration

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,21 @@
+FROM enmasseproject/java-base:11-0
+
```
Comment (lulf) [inline Dockerfile:2]: This should be using quay.io/enmasse/java-base:11-1

File: utils/bin/launch.sh [hunk 2]
```diff
@@ -0,0 +1,60 @@
+#!/bin/sh
+
+source /opt/run-java/dynamic_resources.sh
```
Comment (lulf) [inline utils/bin/launch.sh]: This path should be changed as well. Have a look at latest version in the EnMasse master branch

**Review by lulf — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (k-wall) [reply Dockerfile:2]: Thanks for the catch, I copied from a stale branch in my workspace in error.

File: utils/bin/launch.sh (hunk 2 above)
Comment (k-wall) [reply utils/bin/launch.sh]: Ditto above, thanks.

File: .travis/build.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Assumes that COMMIT, DOCKER_USER and DOCKER_PASS to be set
+VERSION=${1:-"latest"}
+COMMIT=$2
+DOCKER_REGISTRY=quay.io
+REPO=${DOCKER_REGISTRY}/enmasse/qdrouterd-base
```
Comment (lulf) [inline .travis/build.sh:6]: @k-wall wrong repo

**Pull request merged**